cardano-cli/src/Cardano/CLI/Shelley/Run/Genesis.hs

```diff
@@ -636,7 +621,7 @@ runGenesisCreateCardano (GenesisDir rootdir)
                       (toByronRequiresNetworkMagic network)
     byronBalance = TestnetBalanceOptions
         { tboRichmen = genNumGenesisKeys
-        , tboPoors = 1
+        , tboPoors = genNumUTxOKeys
```

Comment:
This could be in a commit on its own, which will make it easier to see that you are fixing the previously ignored "genNumUTxOKeys" argument.

Reply:
That single line would not fix it.
@CarlosLopezDeLara found out that the node rejects the generated genesis files.
(I confirmed it).
I still believe that the original version of `cardano-cli genesis create-cardano` was not working.